modules/machines-edge-machine-pool.adoc

```diff
@@ -0,0 +1,103 @@
+
+:_content-type: CONCEPT
+[id="machines-edge-machine-pool_{context}"]
+= The edge machine pool
+
+{product-title} 4.12 introduced a new compute pool, _edge_, that is designed for use in remote zones. The edge compute pool configuration is common between AWS Local Zone locations. However, due to the image type and size limitations on Local Zone resources, the default instance type that is created can vary from the traditional worker pool.
+
+The default elastic block store (EBS) for Local Zone locations is `gp2`, which differs from the default worker pool. The preferred list of instance types for the edge compute pool follows the same order of worker pools, depending on the availability of the location.
+
+The edge compute pool creates new labels that developers can use to deploy their applications onto those locations. The new labels are:
+
+* `node-role.kubernetes.io/edge=''`
+* `machine.openshift.io/zone-type=local-zone`
+* `machine.openshift.io/zone-group=<$ZONE_GROUP_NAME>`
+
+
+By default, the system creates the edge compute pool manifests only if users add AWS Local Zone subnet IDs to the list `platform.aws.subnets`.
+
+The edge compute pool's machine sets have a `NoSchedule taint` by default to prevent regular workloads from being spread out on those machines. Users can only run user workloads if the tolerations are defined on the pod spec. 
+
+The following examples show `install-config.yaml` files that use the edge machine pool. 
+
+.Configuration that uses an edge pool with default settings
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: devcluster.openshift.com
+metadata:
+  name: ipi-localzone
+platform:
+  aws:
+    region: us-west-2
+    subnets:
+      - subnetID-private1
+      - subnetID-private2
+      - subnetID-private3
+      - subnetID-public1
+      - subnetID-public2
+      - subnetID-public3
+      - subnetID-localzone-public1
+  pullSecret: '{"auths": ...}'
+sshKey: ssh-ed25519 AAAA...
+----
+
+.Configuration that uses an edge pool with a custom instance type
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: devcluster.openshift.com
+metadata:
+  name: ipi-localzone
+compute:
+- name: edge
+  platform:
+    aws:
+      type: m5.4xlarge
+platform:
+  aws:
+    region: us-west-2
+    subnets:
+    - subnet-0fc845d8e30fdb431
+    - subnet-0a2675b7cbac2e537
+    - subnet-01c0ac400e1920b47
+    - subnet-0fee60966b7a93da6
+    - subnet-002b48c0a91c8c641
+    - subnet-093f00deb44ce81f4
+    - subnet-0f85ae65796e8d107
+pullSecret: '{"auths": ...}'
+sshKey: ssh-ed25519 AAAA...
+----
+
+Instance types differ between locations. Check the AWS documentation to verify availability in the Local Zone in which the cluster will run.
+
+.Configuration that uses an edge pool with a custom EBS type
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: devcluster.openshift.com
+metadata:
+  name: ipi-localzone
+compute:
+- name: edge
+  platform:
+    aws:
+      rootVolume:
+        type: gp3
```

Comment:
@mtulio @maxwelldb change to `gp2`? as `gp3` is not supported

Reply:
```suggestion
        type: gp2
```
@mtulio Any objections?

Reply:
@yunjiang29 `gp3`, there are zones that support `gp3`, two specifically: `us-west-2-lax-1{a,b}. It can change over the time, the goal here is to provide an example how to replace the default, `gp2`.

Do you think it may be confusing? I am thinking to aggregate all those examples to keep it simple and standardized with our docs. WDYT?

Reply:
@yunjiang29 @mtulio Any decision on this?

Reply:
Keep `gp3`, it's intentional to keep it here exemplifying how to change the default (`gp2`).
`gp3` is supported in some Locations: https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/

Reply:
> Is it confusing? I am thinking of aggregating all those examples to keep it simple and standardized with our docs. WDYT?

> Keep gp3

@mtulio @maxwelldb  it looks good to me